### ISSUE TITLE
Dense predictions for semantic segmentation (fully convolutional)

### ISSUE THREAD

**Issue by ahundt** [raghakot/keras-resnet#23]
FYI I'm planning on working on converting this to work with dense predictions based on the [resnet v2 in tensorflow models](https://github.com/tensorflow/models/blob/master/slim/nets/resnet_v2.py) via the use of [AtrousConvolutions2D in Keras](https://keras.io/layers/convolutional/#atrousconvolution2d).

It will be in this branch https://github.com/ahundt/keras-resnet/tree/dense_predictions

And I'm hoping to train/test on camvid and coco, some scripts to load camvid are in this [keras segnet implementation](https://github.com/pradyu1993/segnet).

**Comment by raghakot** [comment]
cool

**Comment by bhack** [comment]
@ahundt Are you working on a specific fork?

**Comment by ahundt** [comment]
@bhack yes as linked above https://github.com/ahundt/keras-resnet/tree/dense_predictions should be resuming work a bit this week.

**Comment by Ssica** [comment]
Hi @ahundt, any status on the progress?

**Comment by ahundt** [comment]
Full details are at https://github.com/farizrahman4u/keras-contrib/issues/63
